pkg/rbac/rbac.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2020 Clastix Labs.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rbac
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/equality"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+type RBACManager struct {
+	ClientConfig *rest.Config
+	CapsuleGroup string
+	Log          logr.Logger
+}
+
+func (r RBACManager) SetupCapsuleRoles() error {
+	// Create client as mgr.Client won't be availiable for us until we call mgr.Start
+	k8sClient, err := client.New(r.ClientConfig, client.Options{})
```

Comment:
why importing `k8sClient` here using the `*rest.Config`?
You can use the `client.Client` interface provided by `mgr.GetClient()` in order to use also the local cache.

Reply:
mgr.GetClient() won't be available until manager starts, as it relies on caches. It was my first try to use it - but I won't be able to GET resources due to error

Reply:
I guess we can overcome this with the following [PR](https://github.com/wgsateam/capsule/pull/1)

Reply:
Great catch, thanks! Will merge it right now